include/engine/datafacade/shared_datafacade.hpp

```diff
@@ -68,6 +69,7 @@ class SharedDataFacade final : public BaseDataFacade
     std::unique_ptr<storage::SharedMemory> m_layout_memory;
     std::unique_ptr<storage::SharedMemory> m_large_memory;
     std::string m_timestamp;
+    extractor::ProfileProperties* m_profile_properties;
```

Comment:
Why is this a raw pointer? Who owns it?

/edit oh, I see below this is coming from shared memory.